src/pyprojectx/initializer/initializers.py

```diff
@@ -157,7 +157,10 @@ def initialize_global(options):
         print(
             f"Add the scripts to your PATH, f.e. by appending following to your shell's profile"
             f" ({RESET}~/.profile{BLUE}, {RESET}~/.zshrc{BLUE}, {RESET}~/..bashrc{BLUE}, ...):\n"
-            f"{RESET}export PATH=$PATH:{global_dir.parent.absolute()}, file=sys.stderr"
+            f"{RESET}export PATH=$PATH:{global_dir.parent.absolute()}"
+            f"\nOr for xonsh's ~/.xonshrc append the following\n"
+            f"{RESET}$PATH.append('{global_dir.parent.absolute()}')",
```

Comment:
For this I was just really using something like this which I think should work in both windows and linux but as I hadn't tested it in Windows Xonsh shell yet I left your absolute path there since it should be more reliable.
```
# add pyprojectx path to PATH in ~/.xonshrc
$PATH.append(f'{$HOME}/.pyprojectx')
```
More about Xonsh shell if you were curious.
https://xon.sh/
https://github.com/xonsh/xonsh